zap/src/main/java/org/parosproxy/paros/core/scanner/Alert.java

```diff
@@ -193,6 +195,10 @@ public static Source getSource(int id) {
         "False Positive", "Low", "Medium", "High", "Confirmed"
     };
 
+    private static final String CWE_TEMPLATE_KEY = "CWE-{0}";
+    private static final String CWE_TEMPLATE_URL =
+            "https://cwe.mitre.org/data/definitions/{0}.html";
```

Comment:
It would be simpler to just concatenate. Having hardcoded link in core :/ but guess there's no way around (it other than allowing to set from add-ons).